airbyte-integrations/connectors/source-s3/performance-test-config.yml

```diff
@@ -0,0 +1,11 @@
+performance_tests:
```

Comment:
I would like to avoid this in favor of having metadata entries and doing the slash command wrapper around cloud for performance syncs. I won't block the PR, but I think it's best to do the right thing here. Perhaps a path forward is to keep this local and not have another test config committed.

Reply:
No problem. I was actually removing the performance-test artifacts but missed this particular file. 

For context: I built this in reverse order to what we are reviewing and merging. First wrote a major Polars rewrite, then spun off smaller scope here, then spun out an even smaller scope to get the v5 CDK bump done. The perf tests harness is from several weeks ago now, and we have other ways to test.

What would still be helpful is a semantic language around how we want to define performance tests. My two best bets are (1) add to metadata.yaml or (2) add into the `acceptance-test-config.yaml`. Even if they are a no-op right now, would be helpful to start putting those somewhere in the meanwhile...

cc @alafanechere